data.sql

```diff
@@ -0,0 +1,16 @@
+-- Inserting datas relative to each animals
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+-- Animal: Name, Date of birth, Escape attempts, neutered, weight_kg
+	('Agumon', '2020-02-03', 0, true, 10.23),
+	('Gabumon', '2018-11-15', 2, true, 8.0),
+	('Pikachu', '2021-01-07', 1, false, 15.04),
+	('Devimon', '2017-05-12', 5, true, 11.0),
+	('Charmander', '2020-02-08', 0, false, -11.00),
+	('Plantmon', '2021-11-15', 2, true, -5.7),
+	('Squirtle', '1993-04-02', 3, false, -12.13),
+	('Angemon', '2005-06-12', 1, true, -45.00),
+	('Boarmon', '2005-06-07', 7, true, 20.4),
+	('Blossom', '1998-10-13', 3, true, 17.0),
+	('Ditto', '2022-05-14', 4, true, 22.0);
```

Comment:
- [ ] You have provided in excess of the data you are expected to add to the project requirement. I think it will be better if you stick to the project requirement 👍🏽